radar/infrastructure_ci_cd/puppet.yaml

```diff
@@ -1,5 +1,7 @@
 name: Puppet
 blip:
+  - date: 2019-06-16
```

Comment:
Date should be 2020-06-16 and please add it last in the list. I don't think the radar is smart enough to sort the dates.